README.md

```diff
@@ -126,12 +126,87 @@ If they are not set explicitely, most of them have a default setting that will b
 Required means that the value **must** be set either explicitely or by having an environment variable. 
 If none is provided the `resolve` method call will fail with an exception.
 
-| Property | Default environment variable | Required | Default | Type |
-| -------- | ---------------------------- | -------- | ------- | ---- |
-| `template_location` | `TEMPLATING_TEMPLATE_LOCATION` | Yes | | The location of the template that should be used during the evaluation. If this location starts with either `http` or `https` it will be considered a URL and will be loaded from the specified endpoint over the network. In all other case the location is being considered a file on the local file system. |
-| `template_auth_username` | `TEMPLATING_AUTH_USERNAME` | No | | The username to be used when accessing the template via network using HTTP basic auth. |
-| `template_auth_password` | `TEMPLATING_AUTH_PASSWORD` | No | | The password to be used when accessing the template via network using HTTP basic auth. |
-| `content` |  | Yes | | The content to be used. |
-| `content_placeholder` | `TEMPLATING_CONTENT_PLACEHOLDER` | No | `INCLUDE_CONTENT_HERE` | The placeholder within the template that should be replaced with the content from the service. |
-| `title` | `TEMPLATING_TITLE` | No | `BetterDoc` | The title to be used. |
-| `title_placeholder` | `TEMPLATING_TITLE_PLACEHOLDER` | No | `INCLUDE_TITLE_HERE` | The placeholder within the template that should be replaced with the title. |
+| Property | Default environment variable | Required | Description |
+| -------- | ---------------------------- | -------- | ----------- |
+| `template_location` | `TEMPLATING_TEMPLATE_LOCATION` | Yes | The location of the template that should be used during the evaluation. If this location starts with either `http` or `https` it will be considered a URL and will be loaded from the specified endpoint over the network. In all other case the location is being considered a file on the local file system. |
+| `template_auth_username` | `TEMPLATING_AUTH_USERNAME` | No | The username to be used when accessing the template via network using HTTP basic auth. |
+| `template_auth_password` | `TEMPLATING_AUTH_PASSWORD` | No | The password to be used when accessing the template via network using HTTP basic auth. |
+
+## Dialects
+
+The different dialects have additional configuration options on their own
+
+### `MustacheDialect`
+
+The `MustacheDialect` can either be set explicitely (see below) and will be implicitely selected if no dialect has been set explicitely **and** the template path ends with `.mustache` or `.mustache.html`.
```

Comment:
```suggestion
The `MustacheDialect` can either be set explicitely (see below) and will be implicitely selected if no dialect has been set explicitly **and** the template path ends with `.mustache` or `.mustache.html`.
```

Reply:
Does this mean that if I select the `simple` template engine and refer to `foo.html.mustache`, it will be processed with the simple template engine? I find this somewhat surprising.

Reply:
Yes, that's what will happen.

I don't think it's surprising - if you **explicitely** set the simple dialect then I would expect the template engine to respect that explicit selection (which is what it does).